integration-tests/salesforce/src/main/java/org/apache/camel/quarkus/component/salesforce/SalesforceRoutes.java

```diff
@@ -87,6 +87,12 @@ public void configure() throws Exception {
                     + "updateTopic=true&sObjectQuery=SELECT Id, Name FROM Account")
                             .to("seda:RawPayloadCamelTestTopic");
 
+            // it takes some time for the subscriber to subscribe, so we'll try to
+            // send repeated platform events and wait until the first one is
+            // received
+            from("timer:platform")
+                    .setBody().simple("{}")
```

Comment:
What does `.setBody().simple("{}")` do?

Reply:
@ppalaga is ot okay for you. Do I merge ?

Reply:
Oh, sorry, yes, I approved. This was just a curious question.